test/cql-pytest/test_group_by_limit.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import time
+import pytest
+import logging
+from cassandra.protocol import InvalidRequest, ReadFailure                         # type: ignore
+from cassandra.query import SimpleStatement                                        # type: ignore
+
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_add_index(cql):
+    assert cql is not None
+    cql.execute("create keyspace ks with replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
+    cql.set_keyspace("ks")
+    cql.execute("""create table grouping (
+        a int,
+        b int,
+        c int,
+        PRIMARY KEY (a, b)
```

Comment:
The words case is mixed, sometimes CQL keywords are small letters, sometimes capital letters, please keep everything upper case for simplicity.